.travis.yml

```diff
@@ -30,16 +31,14 @@ install:
 
 before_script:
   - pebble &
-  # Wait for pebble to come up
-  - until </dev/tcp/localhost/14000 ; do sleep 0.1 ; done
```

Comment:
could you explain this change?

Reply:
If Linux has a decent way to poll on a network socket, this is not the case for Windows. In order to have a consistent way to have a uptime check on Pebble on both platform, I moved this process directly on Chisel, making this instruction useless.

As a side note, this migration allows to have a self contained Chisel script that could work on a remote Pebble instance.